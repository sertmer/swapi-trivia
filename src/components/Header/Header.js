import React from 'react'
import './Header.css'

const Header = ({name, rank, quote}) => {
  return (
    <nav>
      <h1>Star Wars Trivia Database</h1>
      <p>{rank} {name}</p>
      <p>{quote}</p>
    </nav>
  )
}

export default Header