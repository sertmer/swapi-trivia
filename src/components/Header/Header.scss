nav {
  display: flex;
  flex-direction: column;

  .buttons_div {
    display: flex;
    flex-direction: row;
    justify-content: space-around;
  }

  h1 {
    font-size: 4em;
  }
}