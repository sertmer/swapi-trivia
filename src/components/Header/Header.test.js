import React from 'react'
import ReactDOM from 'react-dom'
import Header from './Header'
import { shallow } from 'enzyme'

describe('Header', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Header />)
  })

  it('should match the snapshot', () => {   
    expect(wrapper).toMatchSnapshot()
  })

})