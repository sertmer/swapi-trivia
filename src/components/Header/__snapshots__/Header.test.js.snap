// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<nav>
  <h1>
    Star Wars Trivia Database
  </h1>
  <p>
     
  </p>
  <p />
</nav>
`;
